Assets/Scripts/Lever.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class Lever : MonoBehaviour {
+    public UnityEvent leftOption;
+    public UnityEvent rightOption;
+    private bool left;
+    private Animator animator;
+    private bool inRange;
+
+    void Awake()
+    {
+        left = true;
+        inRange = false;
+        animator = gameObject.GetComponent<Animator>();
+    }
+
+    private void Start()
+    {
+        leftOption.Invoke();
```

Comment:
I think this should be in `OnEnable`, and invoke left or right based on the `left` variable.